lib/backbeat/Metrics.js

```diff
@@ -461,6 +461,24 @@ class Metrics {
             });
         });
     }
+
+    /**
+     * Close redis client
+     * @param {function} cb - callback(error, data)
+     * @return {undefined}
+     */
+    disconnect(cb) {
+        return this._redisClient.disconnect(cb);
```

Comment:
I'm thinking we could just have the wrapper functions on RedisClient and remove these methods on Metrics class. Any place we use an instance of this Metrics class, we will also have a RedisClient instance available. So I'm thinking just calling directly to the RedisClient would make more sense